storage/cloud.py

```
@@ -157,17 +161,80 @@ def __init__(
         if deferred_refreshable_credentials:
             self._session._session._credentials = deferred_refreshable_credentials
 
+    def response_tracing_headers(self, **kwargs):
+        name = f"boto3.{str(kwargs.get('event_name', 'before-send.s3.unknown')).split('.')[-1]}"
+        with tracer.start_as_current_span(name) as span:
+            sctx = span.get_span_context()
```

Ruff (0.14.4)

Local variable `sctx` is assigned to but never used

Remove assignment to unused variable `sctx`

(F841)